generator/statistics.hpp

```diff
@@ -33,6 +33,25 @@ namespace stats
     double m_area;
   };
 
+  struct GeomInfo
+  {
+    GeomInfo() : m_count(0), m_size(0), m_elements(0) {}
```

Comment:
You don't need that constructor, just assign zero to each member variable directly.